app/views/leakdetection/LeakDetectionRepositoriesPage.scala.html

```diff
@@ -56,6 +55,21 @@ <h1>Leak Detection Repositories</h1>
                 'id -> "team-dropdown"
             )
             </div>
+            <div class="col-xs-4">
+                <label>Show repositories with:</label>
+                <div>
+                    <input name="includeWarnings" id="includeWarnings" type="checkbox" onchange="form.submit()" value="true"
+                    @if(form("includeWarnings").value == Some("true")) {checked}> Warnings</input>
+                </div>
+                <div>
+                    <input name="includeExemptions" id="includeExemptions" type="checkbox" onchange="form.submit()" value="true"
+                    @if(form("includeExemptions").value == Some("true")) {checked}> Exemptions</input>
+                </div>
+                <div>
+                    <input name="includeViolations" id="includeViolations" type="checkbox" onchange="form.submit()" value="true"
+                    @if(form("includeViolations").value == Some("true")) {checked}> Violations</input>
```

Comment:
`value.exists(_ == "true")` or if we define the form as a Boolean value `value.exists(_)`?